Epsilon/Service/CompetenceDocumentService.cs

```diff
@@ -27,7 +27,7 @@ public async Task<Stream> WriteDocument(Stream stream)
         using var document = WordprocessingDocument.Create(stream, WordprocessingDocumentType.Document);
 
         document.AddMainDocumentPart();
-        document.MainDocumentPart!.Document = _wordDocumentBuilder.Build(components);
+        document.MainDocumentPart!.Document = _wordDocumentBuilder.Build(document.MainDocumentPart, components);
```

Comment:
Why is it needed to change the architecture? Cant you create an open XML element and just return that?

Reply:
This was required due to the fact that we needed to call `mainDocumentPart.AddAlternativeFormatImportPart(AlternativeFormatImportPartType.Html, altChunkId);`. I'll have another look though